### ISSUE TITLE
Icon: Icon border inverse focus accessibility

### ISSUE THREAD

**Issue by interactivellama** [salesforce/design-system-react#1946]
The following SLDS CSS that should be working, but isn't. This is probably be a missing class name issue--if I had to guess
```
.slds-button_inverse:focus,
.slds-button--inverse:focus,
.slds-button_icon-border-inverse:focus,
.slds-button--icon-border-inverse:focus {
  outline: none;
  box-shadow: 0 0 3px #ecebea;
  border: 1px solid #ecebea; }
```

LDSR:
![Screen Shot 2019-04-23 at 12 43 23 PM](https://user-images.githubusercontent.com/1290832/56607301-735dca80-65c5-11e9-9142-b85a5309d045.png)

SLDS:
![Screen Shot 2019-04-23 at 12 44 22 PM](https://user-images.githubusercontent.com/1290832/56607351-96887a00-65c5-11e9-9ffc-5bb53fbc7cce.png)

@jcjuett I think these are the same and the CSS is working

**Comment by interactivellama** [comment]
@jcjuett I'm going to close this issue until we know what we want to do.

@kevinparkerson for awareness that we don't know what we are doing with this issue.

**Comment by jcjuett** [comment]
This makes sense. I'll keep open on my spreadsheet until we come up with a solution.
